src/context/WalletProvider/WalletProvider.tsx

```diff
@@ -212,8 +214,10 @@ const reducer = (state: InitialState, action: ActionTypes) => {
       return {
         ...state,
         modal: true,
+        showBackButton: false,
         type: KeyManager.KeepKey,
         deviceId: action.payload.deviceId,
+        onCloseRoute: '/connect-wallet',
         initialRoute: KeepKeyRoutes.WipeSuccessful,
```

Comment:
We should rename this route now that it'll represent the state of a KeepKey being connected with factory settings, rather than a wiped KeepKey specifically.

```ts
        initialRoute: KeepKeyRoutes.FactoryState,
```

Or something similar.